certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/action/model/ActionEvaluationUtil.java

```diff
@@ -0,0 +1,18 @@
+package se.inera.intyg.certificateservice.domain.action.model;
+
+import java.util.Optional;
+
+public class ActionEvaluationUtil {
```

Comment:
As discuss lets avoid util/toolkits in the domain and instead add logic to the domain objects.